content/en/logs/guide/identify-and-exclude-noisy-logs.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Identify and Exclude Noisy Logs
+kind: guide
+disable_toc: true
+further_reading:
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "Learn more about Log Explorer"
+- link: "logs/explorer/patterns/"
+  tag: "Documentation"
+  text: "Learn about log patterns"
+- link: "logs/live_tail"
+  tag: "Documentation"
+  text: "Explore Live Tail"
+- link: "logs/logs_to_metrics"
+  tag: "Documentation"
+  text: "Learn how to generate metrics from ingested logs"
+---
+
+## Overview
+
+Datadog’s [Logging without Limits][1]&trade; removes logging limitations by decoupling log ingestion and indexing to give flexibility around log management. High volume log indexing is helpful when monitoring every aspect of a service, but it can be a distraction when serious problems occur.
+
+For example, you may not find it beneficial to monitor every 200 response code log from a web server, as you know this is not relevant for troubleshooting. It’s more beneficial to filter these logs so when an issue occurs, you can quickly drill down to the source of the problem and resolve it.
+
+This guide uses Logging without Limits&trade; to identify a noisy logging service and status, exclude irrelevant status logs, and set custom metrics from the excluded logs to continue to track data over time.
+
+## 1. Identify a noisy logging service and status
```

Comment:
We should add screenshots to this section to better demonstrate how to setup a log analytics + the final result of this process.